game/state/gamestate_serialize.cpp~RF3c74b548.TMP

```diff
@@ -0,0 +1,1840 @@
+#include "game/state/gamestate.h"
```

Comment:
I'm just saying this while .TMP file seems like it doesn't need to be committed, and possibly only there due to being left from a cancelled/crashed clang-format run, as I believe that uses similarly named temporary files.

You should be able to remove it by deleting the file, then it should show up as 'removed' in github, so it will be removed in the next commit.

If you fix this and the other things mentioned below in another commit, I can squash-merge in, so no need to worry about fancy git commands.